ci/cisuccessjob.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: "container-pipeline-ci-job"
```

Comment:
Should we name this `container-pipeline-succes-ci-job` since this is checking for success case(s)?